tests/rptest/tests/transaction_kafka_api_test.py

```diff
@@ -28,8 +28,10 @@ def __init__(self, test_context):
                                  "tx_timeout_delay_ms": 10000000,
                                  "abort_timed_out_transactions_interval_ms":
                                  10000000,
-                                 'enable_leader_balancer': False
-                             })
+                                 "enable_leader_balancer": False,
+                                 "transaction_coordinator_partitions": 4
+                             },
+                             log_level="trace")
```

Comment:
do we need to setup log level, i think it is defaulted to trace in ducktape ?

Reply:
we need, default log level is info